src/vs/workbench/contrib/terminal/browser/xterm/cognisantCommandTrackerAddon.ts

```diff
@@ -133,10 +133,10 @@ export class CognisantCommandTrackerAddon extends CommandTrackerAddon {
 						timestamp: Date.now(),
 						cwd: this._cwd,
 						exitCode: this._exitCode,
-						getOutput: () => getOutputForCommand(this._currentCommand.previousCommandMarker!.line! + 1, this._currentCommand.marker!.line!, buffer)
+						getOutput: () => getOutputForCommand(this._currentCommand.previousCommandMarker!.line! + 1, this._currentCommand.marker!.line!, buffer),
```

Comment:
Can getOutputForCommand take the TerminalCommand object so the details are hidden here?